query/keys.go

```diff
@@ -361,3 +362,7 @@ func getAccountParamKey() []byte {
 func getPostParamKey() []byte {
 	return postParamSubStore
 }
+
+func GetReputationParamKey() []byte {
```

Comment:
Is this exported method? where did u use this, are u trying to offer another query method or?

Reply:
done.